testsuite/integration-arquillian/tests/base/src/main/java/org/keycloak/testsuite/arquillian/ServerTestEnricherUtil.java

```diff
@@ -0,0 +1,63 @@
+package org.keycloak.testsuite.arquillian;
```

Comment:
Missing the license text for the new file :-)

Reply:
@miquelsi Ahh, right!

Reply:
@miquelsi Done.